Ozone Project/Ozone/src/test/java/Ozone.java

```diff
@@ -0,0 +1,118 @@
+import org.openqa.selenium.*;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.Wait;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Ozone {
+    WebDriver driver;
+    WebDriverWait wait;
+
+    String filePath = "D:/DiscountedElements.txt";
+    boolean hasNextPage = true;
+    int attempts = 0;
+    int maxAttempts = 3;
+
+    @BeforeMethod
+    public void setup() {
+        driver = new ChromeDriver();
+        driver.manage().window().maximize();
+        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(3));
+        driver.get("https://www.ozone.bg/");
+
+        wait = new WebDriverWait(driver, Duration.ofSeconds(3));
+    }
+
+    @Test
+    public void loginTest() throws InterruptedException {
+//        wait.until(ExpectedConditions.visibilityOfElementLocated(By.xpath("//*[@id=\"cookie_info_agree\"]/a")));
+
+
+        WebElement cookie = driver.findElement(By.xpath("//*[@id=\"cookie_info_agree\"]/a"));
+        cookie.click();
+        Actions builder = new Actions(driver);
+        WebElement category = driver.findElement(By.cssSelector(".open-main-cat-nav-wrapper>a"));
```

Comment:
you can extract this in a method, the whole logic of finding element and moving to it. It can accept variable, can be also array of strings, and it will go through each element, find them and move to them.